internal/table/table.go

```diff
@@ -51,7 +52,11 @@ func (t *Table) Render() string {
 	var out string
 	if !t.HideHeaders {
 		out += "[NOTICE]" + renderRow(t.headers, colWidths) + "[/RESET]" + linebreak
-		out += "[DISABLED]" + strings.Repeat(dash, total) + "[/RESET]" + linebreak
+		if !t.HideDash {
+			out += "[DISABLED]" + strings.Repeat(dash, total) + "[/RESET]" + linebreak
+		} else {
+			out += "[DISABLED]" + strings.Repeat(" ", total) + "[/RESET]" + linebreak
+		}
```

Comment:
This doesn't appear to be working for me:

![image](https://github.com/ActiveState/cli/assets/278649/5d07a8d5-76b4-4ba7-a024-209eda1a838c)

Also, we should probably just not render this line at all rather than make it "empty".

Reply:
Whoops, not sure how that happened. I've updated it not to print the line however the spacing isn't ideal if the line is printed or not.

Empty line:
```
  Name        Version          Vulnerabilities (CVEs)

  python      3.9.13           1 Critical, 7 High, 3 Medium
  libsass     auto → 0.21.0    None detected
  pexpect     4.4.0            None detected
  psutil      auto → 5.9.0     None detected
  requests    2.25.0           1 Medium

For CVE info run 'state security'
```

No line:
```
  Name        Version          Vulnerabilities (CVEs)
  python      3.9.13           1 Critical, 7 High, 3 Medium
  libsass     auto → 0.21.0    None detected
  pexpect     4.4.0            None detected
  psutil      auto → 5.9.0     None detected
  requests    2.25.0           1 Medium

For CVE info run 'state security'
```

Maybe we should revert this and keep the dash?